Cargo.toml

```diff
@@ -128,10 +128,10 @@ tracing = { version = "0.1", features = ["log"] }
 wasmtime-wasi = { version = "7.0.0", features = ["tokio"] }
 wasi-common-preview1 = { package = "wasi-common", version = "7.0.0" }
 wasmtime = { version = "7.0.1", features = ["component-model"] }
-spin-componentize = { git = "https://github.com/fermyon/spin-componentize" }
-wasi-host = { package = "host", git = "https://github.com/fermyon/spin-componentize" }
-wasi-common = { git = "https://github.com/fermyon/spin-componentize" }
-wasi-cap-std-sync = { git = "https://github.com/fermyon/spin-componentize" }
+spin-componentize = { git = "https://github.com/fermyon/spin-componentize", rev = "9e7ec95ccaf490c9b96c03af1c818e8197ccf855" }
```

Comment:
lgtm! 

nit: create a git tag to spin-componentize repo and then pin to this specific tag could make Cargo.toml much more readable.

Reply:
I agree though we're iterating a bit currently. Once it settles, I think we can tag a release that's meant for the next version of spin.